### PR TITLE
Set guidance on handling AFTOperation errors

### PR DESCRIPTION
**Pull request by xw-g**
Propose a general guidance on server handling AFTOperation processing errors.

### CODE REVIEW

File: v1/proto/service/gribi.proto [hunk 1]
```diff
@@ -52,8 +52,12 @@ service gRIBI {
 message ModifyRequest {
   // A group of requests to add/modify/remove a single AFT entry.
   // gRIBI server relies on clients being conservative in what they send.
-  // As such, gRIBI server is expected to process the AFTOperations per the
-  // received order.
+  // As such, when processing AFTOperations, a gRIBI server is expected to:
+  //  * process AFTOperations per the received order.
+  //  * try not to close the RPC session due to errors in AFTOperation
```
Comment (robshakir) [inline v1/proto/service/gribi.proto:57]: "try not to" vs. "not" -- do we want to be stronger here?

I think that in general the statement here should probably be:

```
errors that are encountered in a single AFTOperation should not
result in the Modify RPC session being closed. Rather, invalid
AFTOperations should be responded to with a failure within the
stream.
```

File: v1/proto/service/gribi.proto [hunk 2]
```diff
@@ -52,8 +52,12 @@ service gRIBI {
...
+  //    processing (e.g. invalid network instance, unsupported op, etc), unless
+  //    it’s considered fatal to the device (e.g. might crash the whole device
+  //    routing daemon).
```
Comment (robshakir) [inline v1/proto/service/gribi.proto:60]: Are there any such cases that would crash the daemon that we can know about? I feel like NPE might be one, but this is something that we could pre-check for (i.e., check that the operation != nil - if it does, return an error directly).

**Review by robshakir — CHANGES_REQUESTED** [review]

File: v1/proto/service/gribi.proto (hunk 2 above)
Comment (xw-g) [reply v1/proto/service/gribi.proto:61]: I think this is a good point. Changed this point to use stronger word. Please help take a look again. Thanks.

File: v1/proto/service/gribi.proto (hunk 1 above)
Comment (xw-g) [reply v1/proto/service/gribi.proto:58]: ack. make sense.

**Pull request merged**